src/storage/txn/commands/atomic_store.rs

```diff
@@ -20,7 +20,7 @@ command! {
             /// The set of mutations to apply.
             cf: CfName,
             mutations: Vec<Mutation>,
```

Comment:
Please define a `RawMutation` that contains:

- Put(key, value, ttl)
- Delete(key)

Reply:
This should solve a few problems above

Reply:
> Please define a `RawMutation` that contains:
> 
> * Put(key, value, ttl)
> * Delete(key)

committed

Reply:
and how can I test the forcas switch? To modify code of client-go?